### PR TITLE
doc: Adds trobuleshooting guidance when IAM Role has been deleted

### PR DESCRIPTION
**Pull request by EspenAlbert**
## Proposed changes

Adds trobuleshooting guidance when IAM Role has been deleted

Link to any related issue(s): 

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as
  expected)
- [ ] This change requires a documentation update
- [ ] If changes include removal or addition of 3rd party GitHub actions, I updated our internal document. Reach out to the APIx Integration slack channel to get access to the internal document.

## Manual QA performed:

- [ ] cfn invoke for each of CRUDL/cfn test
- [ ] Updated resource in  [example](https://github.com/mongodb/mongodbatlas-cloudformation-resources/tree/master/examples)
- [ ] Published to AWS private registry
- [ ] Used the template in [example](https://github.com/mongodb/mongodbatlas-cloudformation-resources/tree/master/examples) to create and update a stack in AWS
- [ ] Deleted stack to ensure resources are deleted
- [ ] Created multiple resources in same stack
- [ ] Validated in Atlas UI
- [ ] Included screenshots

## Required Checklist:

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that this change does not generate any credentials and that **they are NOT accidentally logged anywhere**.
- [ ] I have added any necessary documentation (if appropriate)
- [ ] I have run `make fmt` and formatted my code
- [ ] For CFN Resources: I have released by changes in the private registry and proved by change
  works in Atlas

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -147,6 +152,15 @@ Use the [execution-role.yaml](examples/execution-role.yaml) file to generate an
 Alternatively, you can set the trust relationships of your role via AWS Console: in the IAM dashboard, select your role and click **Trust Relationships**:
 ![Screenshot 2023-03-31 at 17 32 55](https://user-images.githubusercontent.com/5663078/230436500-fb4ee057-b70e-4580-a94d-f56191728117.png)
 
+### Activate Type IAM Role deleted
+The IAM Role used to Activate the resource type might have been deleted (also applies when using a different IAM Role when creating the stack).
+To verify this you can:
+
+1. ![alt text](docs/images/activate_type_iam_role1.png)
```
Comment (lantoli) [inline README.md:159]: knit: some real alt text :-)

File: README.md [hunk 2]
```diff
@@ -147,6 +152,15 @@ Use the [execution-role.yaml](examples/execution-role.yaml) file to generate an
 Alternatively, you can set the trust relationships of your role via AWS Console: in the IAM dashboard, select your role and click **Trust Relationships**:
 ![Screenshot 2023-03-31 at 17 32 55](https://user-images.githubusercontent.com/5663078/230436500-fb4ee057-b70e-4580-a94d-f56191728117.png)
```
Comment (lantoli) [inline README.md:153]: nit: i know it's not change in this PR, but better change the "screenshot..."

**Review by lantoli — APPROVED** [review]
LGTM, very nice!

**Comment by marcosuma** [comment]
Looks great! thank you for taking the time for this.

**Review by marcosuma — APPROVED** [review]
+1 to Leo's comment on adding some real alt-text, rest is LGTM!

File: README.md (hunk 1 above)
Comment (EspenAlbert) [reply README.md]: Thanks for the comment, updated in 480ba7440fa549757773ca511ffa111ee5a2de56 ✅

File: README.md (hunk 2 above)
Comment (EspenAlbert) [reply README.md]: Thanks for the comment, updated in 480ba7440fa549757773ca511ffa111ee5a2de56 ✅

**Pull request merged**